core/packages/test/scripts/set-env.sh

```diff
@@ -37,42 +37,9 @@ address_for()
     jq -r ".contracts.${1}.address" "$output_dir/contracts.json"
 }
 
-kill_trap() {
-    trap - SIGTERM
-    pkill -P $$
-}
-
 kill_all() {
-    kill_trap
-    kill_chains
-    kill_relayer
-}
-
-kill_chains() {
-    echo "Killing chains"
-    kill_polkadot
-    kill_ethereum
-}
-
-kill_ethereum() {
-    pkill -9 -f lodestar
-    pkill -9 geth
-}
-
-kill_polkadot() {
-    pkill -9 polkadot
-    pkill -9 snowbridge-test-node
-    pkill -9 snowbridge
-    pkill -9 -f polkadot-launch
-    pkill -9 zombienet
-}
-
-kill_relayer() {
-    echo "Killing relayer"
-    kill_trap
-    sleep 3
-    pkill -9 snowbridge-relay
-    sleep 1
+    trap - SIGTERM
+    kill 0
```

Comment:
@yrong I changed the kill behavior here to be a bit more simpler. `kill 0` will kill the whole process group of the current process. I verified that this works on MacOS, and should also work on Linux too (will test).